apps/main/src/app/Home/Home.tsx

```diff
@@ -1,375 +1,121 @@
-└── README.md                # This file`}
-          </code>
-        </pre>
-      </section>
-
-      {/* Available Scripts Section */}
-      <section className="mb-8">
-        <h2 className="text-2xl font-bold mb-4">Available Scripts</h2>
-        <ul className="list-disc list-inside space-y-2">
-          <li><code>start:main</code>: Starts the frontend in development mode (without Docker).</li>
-          <li><code>build:main</code>: Builds the frontend for production.</li>
-          <li><code>test:main</code>: Runs tests for frontend apps.</li>
-          <li><code>lint:main</code>: Lints the monorepo project defined in scripts.</li>
-          <li><code>e2e:main</code>: Runs e2e tests for monorepo project defined in scripts.</li>
-          <li><code>start:backend</code>: Starts the Express server.</li>
-        </ul>
-      </section>
+      {/* Glavna vsebina */}
+      <div className="flex-1 ml-64">{renderContent()}</div>
+    </div>
+  );
+}
 
-      {/* Docker Configuration Section */}
-      <section className="mb-8">
-        <h2 className="text-2xl font-bold mb-4">Docker Configuration</h2>
-        <p className="mb-4">
-          The <code>docker-compose.yml</code> file defines the following services:
-        </p>
-        <ul className="list-disc list-inside space-y-2">
-          <li><strong>backend</strong>: Express app served on port 8000.</li>
-          <li><strong>mariadb</strong>: MariaDB database.</li>
-          <li><strong>redis</strong>: Redis server for caching.</li>
-        </ul>
-        <p className="mt-4">
-          You can customize the Docker configuration by editing the <code>docker-compose.yml</code> file or the
-          individual Dockerfiles.
-        </p>
-      </section>
+// Komponenta Sidebar (meni)
+function Sidebar({ activeTab, setActiveTab }: any) {
+  const MENU_ITEMS = [
+    { name: 'Koledar', icon: CalendarIcon },
+    { name: 'Pregled strank', icon: UserGroupIcon },
+    { name: 'Analitika', icon: ChartBarIcon }
+  ];
+
+  const SETTINGS_ITEMS = [
+    { name: 'Lokacija', icon: MapPinIcon },
+    { name: 'Storitve', icon: HeartIcon },
+    { name: 'Osebje', icon: UsersIcon },
+    { name: 'Urniki', icon: ClockIcon },
+    { name: 'Splošne nastavitve', icon: Cog6ToothIcon },
+    { name: 'Obvestila', icon: BellIcon }
+  ];
 
-      {/* Production Deployment Section */}
-      <section className="mb-8">
-        <h2 className="text-2xl font-bold mb-4">Production Deployment</h2>
-        <p className="mb-4">
-          For production deployment, this project includes a <code>docker-compose.prod.yml</code> file optimized for
-          running the REMR Stack in a production environment. This configuration ensures that all services (frontend,
-          backend, MariaDB, and Redis) are built and run with production-ready settings.
-        </p>
+  return (
+    <div className="w-64 bg-white h-screen fixed left-0 top-15 border-r border-gray-200 flex flex-col shadow-lg">

```

Comment:
```suggestion
    <div className="w-64 bg-white h-svh left-0 border-r border-gray-200 flex flex-col shadow-lg">

```
tle lahku probas zamenat h-screen na h-svh da ti bo dalu cez celo kar je frej. nism 100%